### PR TITLE
WINUI-3163-Added proper redirect link

### PR DESCRIPTION
**Pull request by MoneeshramDhanabal**
Added proper UG redirect links for the DataTimeRangeNavigator control.

### CODE REVIEW

File: xamarin-android/SfDateTimeRangeNavigator/tooltip.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-description: Tooltip customization
+title: Tooltip customization for DateTimeRangeNavigator | Syncfusion
+description: Learn about how to show the selected range start and end value of the Xamarin.Android SfDateTimeRangeNavigator.
 platform: Xamarin.Android
 control: SfDateTimeRangeNavigator
 documentation: ug
 ---
 
-# Tooltip
+# Tooltip in Xamarin.Android SfDateTimeRangeNavigator
```
Comment (MuneeshKumarG) [inline xamarin-android/SfDateTimeRangeNavigator/tooltip.md:11]: Tooltip in Xamarin.Android SfDateTimeRangeNavigator -> Tooltip in Xamarin.Android RangeNavigator.

Update in all sections

Comment (MoneeshramDhanabal) [reply xamarin-android/SfDateTimeRangeNavigator/tooltip.md:11]: Updated the mentioned feedback in all sections.

**Comment by MoneeshramDhanabal** [comment]
rebuild

**Review by MuneeshKumarG — APPROVED** [review]
This is fine

**Pull request merged**